canvas_fe/src/components/users/list.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { Table, Button } from "reactstrap";
+import UserProfileModal from "./profileModal";
+
+class UserList extends Component {
+    constructor() {
+        super();
+        this.state = {isOpen: false};
+    }
+    openModal = (user_id) => {
+        this.setState({isOpen: true});
+        return <UserProfileModal isOpen={this.state.isOpen} />;
```

Comment:
1. put UserProfileModal below </Table>
2. only render when isOpen is true i.e. isOpen ? <UserProfileModal /> : null